Broken Links Found: Moonbeam

### 🚨 Link Checker Report
  **Project:** Moonbeam
  **Date:** 2026-06-18
  **Workflow Run:** https://github.com/papermoonio/workflows/actions/runs/27762316646

  ---
  # Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 38436 |
| ✅ Successful | 37858 |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 577   |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 1     |

## Errors per input

### Errors in moonbeam-mkdocs/site/builders/ethereum/json-rpc/debug-trace/index.html

* [ERROR] [https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool#txpool-status](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool#txpool-status) | Network error: error sending request for url (https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool#txpool-status)
[Full Github Actions output](https://github.com/papermoonio/workflows/actions/runs/27762316646?check_suite_focus=true)

--- a/.urlignore
+++ b/.urlignore
@@ -14,6 +14,7 @@ https://x.com/
 
 # Ignore known working URLs
 https://ethereum.org/
+https://geth.ethereum.org/
 https://moonbase-minterc20.netlify.app/
 https://moonbeam-explorer.netlify.app/
 

